src/coreclr/tools/aot/ObjWriter/llvm.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/include/llvm/BinaryFormat/COFF.h b/include/llvm/BinaryFormat/COFF.h
-index b395db6eaa8..74820a8f79f 100644
```

Comment:
I don't know why these changed. I ran `git diff --output c:\git\runtime\src\coreclr\tools\aot\ObjWriter\llvm.patch`

Reply:
you may want to try `--abbrev=11`

Reply:
Whoa, magic! Thanks!